ext/node/ops/sqlite/database.rs

```diff
@@ -94,27 +101,32 @@ fn open_db(
   state: &mut OpState,
   readonly: bool,
   location: &str,
+  allow_extension: bool,
 ) -> Result<rusqlite::Connection, SqliteError> {
+  let perms = state.borrow::<PermissionsContainer>();
   if location == ":memory:" {
     let conn = rusqlite::Connection::open_in_memory()?;
     assert!(set_db_config(
       &conn,
       SQLITE_DBCONFIG_ENABLE_ATTACH_WRITE,
       false
     ));
-    assert!(set_db_config(
-      &conn,
-      SQLITE_DBCONFIG_ENABLE_LOAD_EXTENSION,
-      false
-    ));
+
+    if allow_extension {
+      perms.check_ffi_all()?;
```

Comment:
Why is there no API name here in case perm check fails?

Reply:
Doesnt seem to be the part of the the FFI perms API.